### ISSUE TITLE
How to install cargo-prusti into cargo?

### ISSUE THREAD

**Issue by rlaferla** [viperproject/prusti-dev#1400]
The **MIRAI** project uses `cargo install --locked --path ./checker` to install into `cargo`.  e.g.  `cargo mirai ` The **Kani** project does similar.  e.g.  `cargo kani` I don't see any documentation on how to do this for **Prusti**.

**Comment by Aurel300** [comment]
I don't think we have tried the `install` command yet. Maybe this would run into issues with the Prusti binary not being able to determine the location of the downloaded Viper tools.

However, you can include the downloaded Prusti package (or the `target/debug` or `target/release` directory in case of source builds) in your `PATH`, in which case `cargo prusti` will detect the `cargo-prusti` binary.
